### PR TITLE
Fix multiline

### PR DESCRIPTION
**Pull request by 3rmack**

### CODE REVIEW

File: shaper/libs/parser.py [hunk 1]
```diff
@@ -260,6 +267,12 @@ def write(self, data, path):
 
 class PropertyParser(TextParser):
 
+    @staticmethod
+    def __process_multiline_string(string):
+        if len(string.splitlines()) > 1:
```
Comment (Alex-Bogdanov) [inline shaper/libs/parser.py]: it's better to init additional variable for `string.splitlines()`

File: shaper/libs/parser.py [hunk 2]
```diff
@@ -176,6 +176,13 @@ def write(self, data, path):
                 represent_unicode,
             )
 
+        # string representer for multi line issue
+        def str_presenter(dumper, data):
```
Comment (Alex-Bogdanov) [inline shaper/libs/parser.py]: I would suggest to move this function to `shaper.libs.loader.py` module

File: shaper/libs/parser.py [hunk 3]
```diff
@@ -176,6 +176,13 @@ def write(self, data, path):
...
+            if len(data.splitlines()) > 1:  # check for multi line string
```
Comment (Alex-Bogdanov) [inline shaper/libs/parser.py]:
```
style = None
if len(data.splitlines()) > 1:  # check for multi line string
     style='|'

return dumper.represent_scalar('tag:yaml.org,2002:str', data, style=style)
```
is much more better

File: shaper/libs/parser.py [hunk 4]
```diff
@@ -260,6 +267,12 @@ def write(self, data, path):
 
 class PropertyParser(TextParser):
 
+    @staticmethod
+    def __process_multiline_string(string):
```
Comment (Alex-Bogdanov) [inline shaper/libs/parser.py]: pls don't use the double underscore, it is something like overhead and java-like style) a single underscore would be fair enough to understand that this method should be used only within the `PropertyParser` class body

**Review by Alex-Bogdanov — CHANGES_REQUESTED** [review]

File: shaper/libs/parser.py (hunk 1 above)
Comment (3rmack) [reply shaper/libs/parser.py]: done

File: shaper/libs/parser.py (hunk 2 above)
Comment (3rmack) [reply shaper/libs/parser.py]: done

File: shaper/libs/parser.py (hunk 3 above)
Comment (3rmack) [reply shaper/libs/parser.py]: done

File: shaper/libs/parser.py (hunk 4 above)
Comment (3rmack) [reply shaper/libs/parser.py]: done

**Review by Alex-Bogdanov — APPROVED** [review]

**Pull request merged**